### PR TITLE
Indent is off by one and snippet check is currently inverse

### PR DESCRIPTION
**Pull request by blakeembrey**

### CODE REVIEW

**Comment by dmartinezg** [comment]
Add test for regression

**Comment by blakeembrey** [comment]
Sorry for the delay, I just went ahead and fixed a few more things with the error message output while testing. Namely:

* Length of snippet was off by one when an odd number
* Indent was off by one (original PR)
* Non-printable characters messed up the error message carat offset in display
* Snippet check is no longer inverted (original PR)
* No more duplicate message in `Error#toString` from same context and message
* Non-printable error message column was off by one

File: test/specs/errors.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ if (typeof window === 'undefined') {
   chai.should();
 }
 
-describe('Errors', function () {
+describe.only('Errors', function () {
```
Comment (dmartinezg) [inline test/specs/errors.js]: `.only()`

**Comment by dmartinezg** [comment]
Remove the '.only()' and merge

**Comment by blakeembrey** [comment]
Removed and I added a timeout to the remote call test (was failing otherwise, at least locally). Should remove that later so we're not hitting a random insurance company servers.

**Pull request merged**